web/src/LocalStorage.tsx

```diff
@@ -33,22 +33,35 @@ export function accessorsForTesting<S>(name: string) {
   }
 }
 
-// Like `useState`, but backed by localStorage and namespaced by the tiltfileKey
-// maybeUpgradeSavedState: transforms any state read from storage - allows, e.g., filling in default values for
-//                         fields added since the state was saved
+// Like `useState`, but backed by localStorage
+// options:
+// - maybeUpgradeSavedState: transforms any state read from storage - allows, e.g., filling in default values for
+//                           fields added since the state was saved
+// - keyedByTiltfile: this state is saved per tiltfile - defaults to true
```

Comment:
isn't there already a pattern for `global` tilt settings used in the font settings? i'm not opposed to configuring this hook to key globally, but i do find the implementation difficult to read.

Reply:
so there is! yeah that seems a lot simpler